api/db/services/user_service.py

```diff
@@ -190,7 +189,7 @@ def decrease(cls, user_id, num):
     @DB.connection_context()
     def user_gateway(cls, tenant_id):
         hashobj = hashlib.sha256(tenant_id.encode("utf-8"))
-        return int(hashobj.hexdigest(), 16)%len(MINIO)
```

Comment:
I'm not quite sure the purpose of referencing MINIO here, it seems like this function is not used yet.